distributed/client.py

```diff
@@ -3864,6 +3876,19 @@ def _get_task_stream(
         else:
             raise gen.Return(msgs)
 
+    def open_dashboard(self):
+        """
+        Open the distributed dashboard.
```

Comment:
```suggestion
        Open the dashboard.
```

I'm trying to remove the name `distributed` from things.  I think that it's not informative for new users.